lib/engine/game/g_18_esp/game.rb

```diff
@@ -1143,6 +1143,20 @@ def start_merge(corporation, minor, keep_token)
 
           # close corp
           close_corporation(minor)
+
+          # delete mz slot if mz
+          fix_cheater_slot_madrid(minor.city) if minor.id == 'MZ'
+        end
+
+        def fix_cheater_slot_madrid(slot)
+          cities = hex_by_id('F24')&.tile&.cities
+          return unless cities
```

Comment:
You don't need to be so defensive. Something is terminally wrong if `hex_by_id('F24').tile.cities` doesn't return an array.